index.md

```diff
@@ -1,28 +1,26 @@
 ---
-layout: page
-title: Extra Unix Shell Material
+layout: lesson
+root: .
 ---
 
 Software Carpentry's [Introduction to the Unix Shell](http://swcarpentry.github.io/shell-novice/) describes the history of the shell, basic commands, and some of the philosophy that makes it such a powerful tool.  This lesson aims to supplement the material in that lesson with short topical lessons on particular shell features that are useful to researchers.  
 
-> ## Prerequisites{.prereq}
+> ## Prerequisites
 >
 > All of this material assumes that learners have mastered
 > the basic Unix shell lesson.  
+{: .prereq}
 
-## Topics
-
-1. [Navigating man pages](01-man-pages.html)
-2. [Connecting to remote machines](02-ssh.html)
-3. [Transferring files](03-file-transfer.html)
-4. [File permissions](04-permissions.html)
-5. [Directory structure](05-directory-structure.html)
-6. [Job control](06-job-control.html)
-7. [Aliases and the bashrc](07-aliases.html)
-8. [Environment variables](08-environment-variables.html)
```

Comment:
> Was this intentionally removed?

Probably, because it's now auto-generated:

* `index.md` [uses the `lesson` layout][1].
* The lesson layout [includes the syllabus][2].
* The syllabus is [compiled (by Jekyll) from the episodes][3].

[1]: https://github.com/swcarpentry/lesson-example/blame/dbcd51d135cc2f17b4d870e3af856c8ea7e1a95b/index.md#L2
[2]: https://github.com/swcarpentry/lesson-example/blob/dbcd51d135cc2f17b4d870e3af856c8ea7e1a95b/_layouts/lesson.html#L6
[3]: https://github.com/swcarpentry/lesson-example/blob/dbcd51d135cc2f17b4d870e3af856c8ea7e1a95b/_includes/syllabus.html